docs/simulators/DualSPHysics.md

```diff
@@ -48,6 +52,395 @@ task.download_outputs()
+if [ $? -ne 0 ] ; then fail; fi
+
+${dualsphysicscpu} ${dirout}/${name} ${dirout} -dirdataout data -svres
+if [ $? -ne 0 ] ; then fail; fi
+
+fi
+
+if [ $option -eq 2 -o $option -eq 1 ]; then
+export dirout2=${dirout}/particles
+${partvtk} -dirin ${diroutdata} -savevtk ${dirout2}/PartFloating -onlytype:-all,+floating
+if [ $? -ne 0 ] ; then fail; fi
+
+${partvtk} -dirin ${diroutdata} -savevtk ${dirout2}/PartFluid -onlytype:-all,+fluid
+if [ $? -ne 0 ] ; then fail; fi
+
+${partvtkout} -dirin ${diroutdata} -savevtk ${dirout2}/PartFluidOut -SaveResume ${dirout2}/_ResumeFluidOut
+if [ $? -ne 0 ] ; then fail; fi
+
+export dirout2=${dirout}/boundary
+${boundaryvtk} -loadvtk AutoActual -motiondata ${diroutdata} -savevtkdata ${dirout2}/Turbine -onlytype:floating -savevtkdata ${dirout2}/Bound.vtk -onlytype:fixed
+
+export dirout2=${dirout}/surface
+${isosurface} -dirin ${diroutdata} -saveiso ${dirout2}/Surface 
+if [ $? -ne 0 ] ; then fail; fi
+
+fi
+if [ $option != 3 ];then
+ echo All done
+ else
+ echo Execution aborted
+fi
+
+read -n1 -r -p "Press any key to continue..." key
+```
+
+As you can see above, the main simulation steps involve these two commands:
+```bash
+# CODES are executed according the selected parameters of execution in this testcase
+${gencase} ${name}_Def ${dirout}/${name} -save:all
+if [ $? -ne 0 ] ; then fail; fi
+
+${dualsphysicscpu} ${dirout}/${name} ${dirout} -dirdataout data -svres
+if [ $? -ne 0 ] ; then fail; fi
+```
+
+The post-processing commands follow, with calls to `partvtk`, `partvtkout`,
+`boundaryvtk` and `isosurface`. These are the commands that generate the data
+required for producing the visualization shown above with ParaView.
+
+There are also a number of variables being set in the beginning of the script. 
+Some of them have to do with the path to DualSPHysics commands, and they
+are assuming a local instalation. For our Inductiva python script, we won't 
+need any of that, since the API takes care of all command paths for us in the
+remote machine. There are also a number of variable related to filenames and 
+input/output directories:
+```bash
+export name=CaseTurbine
+export dirout=${name}_out
+export diroutdata=${dirout}/data
+```
+For a matter of convenience, we will want to keep the control over input / 
+output directories, so we will convert and keep these variables in our python
+script. We will also take the chance to explicitly add a few more output 
+directories, and we will make everything a bit more pythonic. So, in our python script, we should have something like this: 
+```python
+# Let's keep the original variables for the directory names, but we will
+# make them more pythonic and we will add a few others for readibility
+name="CaseTurbine"
+dirout=f"{name}_out"
+data_dirout=f"{dirout}/data"
+particles_dirout=f"{dirout}/particles"
+boundary_dirout=f"{dirout}/boundary"
+surface_dirout=f"{dirout}/surface"
+```
+
+Now, we need to convert the 7 command lines to strings, while making use of
```

Comment:
Next, we need to convert the seven command lines into strings, while making use of the variables we just defined. We’ll also be able to call the DualSPHysics commands (such as gencase, dualsphysics, partvtk, etc.) directly, as they are all pre-installed on the machine we will spin up later.

Reply:
Done!